cmd/manager/main.go

```diff
@@ -124,7 +124,11 @@ func main() {
 	pflag.Parse()
 
 	logs.InitLogs()
-	defer logs.FlushLogs()
+	exitCode := 0
```

Comment:
exitAfterDefer: os.Exit will exit, and `defer logs.FlushLogs()` will not run (gocritic)